### ISSUE TITLE
// vs /**/

### ISSUE THREAD

**Issue by xahon** [eslint/eslint#9294]
**Tell us about your environment**

* **ESLint Version:** 3.19.0
* **Node Version:** 7.10.1
* **npm Version:** 4.2.0

**What parser (default, Babel-ESLint, etc.) are you using?**
babel-eslint 7.2.3

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
module.exports = {
  "parser": "babel-eslint",
  "env": {
    "browser": true
  },
  "plugins": [
    "react",
    "react-native"
  ],
  "extends": [
    "eslint:recommended",
    "plugin:react/recommended",
    "plugin:react-native/all",
    "standard"
  ],
  "parserOptions": {
    "ecmaFeatures": {
      "jsx": true
    }
  },
  "rules": {
    "no-mixed-operators": 0,
    "operator-linebreak": 0
  }
};
```

</details>

**What did you do? Please include the actual source code causing the issue.**

Example that dont work
```js
// eslint no-console: 0
console.log('something')
```
Example that works
```js
/* eslint no-console: 0 */
console.log('something')
```

**What did you expect to happen?**
I expected that // will work same as /**/ because it's comment too

**What actually happened? Please include the actual, raw output from ESLint.**
Eslint error: no-console

**Comment by not-an-aardvark** [comment]
Thanks for the report. However, this is working as intended -- only block comments can be used to configure rules like this.

**Comment by xahon** [comment]
But i don't understand what is the difference and what is reason of such idea? Other cases I can declare like: `// eslint-disable-line`, but here i can't. I think it isn't user-friendly and should be permitted to declare in this way

**Comment by not-an-aardvark** [comment]
`// eslint-disable-line` can only be declared with a line comment and not a block comment, since it only makes sense for it to cover a single line. With all directive comments, I think there's a general goal of only having one way to do it.

The original decision was made in https://github.com/eslint/eslint/pull/320#r7234133.

**Comment by not-an-aardvark** [comment]
Closing because this is working as intended.

**Comment by xahon** [comment]
unintuitive
